config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
       , 'es6/Reflect': 'bower_components/harmony-reflect/reflect'
       , 'codemirror': 'bower_components/codemirror'
       , 'jszip': 'bower_components/jszip/dist/jszip'
+      , 'filesaver': 'bower_components/file-saver.js/FileSaver'
```

Comment:
I see. Well... saveAs will not be used at all by nodeJS. So how should we proceed, then?